tests/e2e/driver_tests.py

```diff
@@ -288,6 +288,20 @@ def test_get_columns(self):
                 for table in table_names:
                     cursor.execute('DROP TABLE IF EXISTS {}'.format(table))
 
+    def test_escape_single_quotes(self):
```

Comment:
this is great, probably want the same type of test for a string with a backslash in it, as in 

```python
test_str = "Many Linux users are upset that Windows uses \\ to separate path elements"
```